### PR TITLE
fix(asciidoc): workaround wrong Maven pom xml display

### PR DESCRIPTION
**Pull request by educhastenier**
Works around issue https://github.com/asciidoctor/asciidoctor/issues/3128

closes [RUNTIME-969](https://bonitasoft.atlassian.net/browse/RUNTIME-969)
closes [RUNTIME-1652](https://bonitasoft.atlassian.net/browse/RUNTIME-1652)

[RUNTIME-969]: https://bonitasoft.atlassian.net/browse/RUNTIME-969?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RUNTIME-1652]: https://bonitasoft.atlassian.net/browse/RUNTIME-1652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/ROOT/pages/customize-living-application-theme.adoc [hunk 1]
```diff
@@ -134,7 +134,7 @@ The maven descriptor is responsible for running the npm build and package the re
 ----
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	xsi:schemaLocation="\http://maven.apache.org/POM/4.0.0 \http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (tbouffard) [inline modules/ROOT/pages/customize-living-application-theme.adoc]: **issue (blocking):** the page displays extra `\`

image

File: modules/ROOT/pages/event-handlers.adoc [hunk 2]
```diff
@@ -61,10 +61,9 @@ pom.xml :
 // for the 'subs' parameter, see https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/
 [source,xml,subs="+macros"]
 ----
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tbouffard) [inline modules/ROOT/pages/event-handlers.adoc:64]: **question:** why removing the xml directive?

Comment (educhastenier) [reply modules/ROOT/pages/event-handlers.adoc:64]: it was a test that I needed to revert and I forgot.
Will revert it.

File: modules/ROOT/pages/customize-living-application-theme.adoc (hunk 1 above)
Comment (educhastenier) [reply modules/ROOT/pages/customize-living-application-theme.adoc]: reverted for this file, but I do NOT understand why we don't have the same problem here 😞 .

File: modules/ROOT/pages/event-handlers.adoc (hunk 2 above)
Comment (educhastenier) [reply modules/ROOT/pages/event-handlers.adoc:64]: done

File: modules/ROOT/pages/custom-authorization-rule-mapping.adoc [hunk 3]
```diff
@@ -37,7 +37,7 @@ In this example, Custom authorization rule is a maven-based java project that ne
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (tbouffard) [inline modules/ROOT/pages/custom-authorization-rule-mapping.adoc:37]: **thought:** I have just discovered that we have the issue only when `<?xml version="1.0" encoding="UTF-8"?>` is present before the line where we see the issue.
I have detected this after you revert a change including the workaround that is not working: https://github.com/bonitasoft/bonita-doc/pull/2324#pullrequestreview-1301658283

Did you try to remove the XML directive to see if this doesn't fix the problem without the need of the workaround?

Comment (educhastenier) [reply modules/ROOT/pages/custom-authorization-rule-mapping.adoc:37]: Yes, that was my motivation, but it did not seem to be enough to get rid of the problem.

Comment (tbouffard) [reply modules/ROOT/pages/custom-authorization-rule-mapping.adoc:37]: OK, fine. 😸

**Review by tbouffard — APPROVED** [review]
LGTM, good job for a painful issue 😈

**Pull request merged**